Signum.Entities.Extensions/Workflow/WorkflowActivity.cs

```diff
@@ -14,7 +14,9 @@ namespace Signum.Entities.Workflow
     [Serializable, EntityKind(EntityKind.Main, EntityData.Master)]
     public class WorkflowActivityEntity : Entity, IWorkflowNodeEntity, IWithModel
     {
-        
+        public static Action<WorkflowActivityEntity, ModelEntity> OnGetModel;

```

Comment:
You could avoid this extension point and override  CopyMixins methods (method name not accurate)